README.md

```diff
@@ -63,6 +63,129 @@ async def main():
 asyncio.run(main())
 ```
 
+## Manipulating lists with `batch_update_list`
+
+This method uses the newer API endpoint for adding, completing and removing items from a list, which is also used in the Bring App. The items can be identified by their uuid and therefore some things are possible that are not possible with the legacy endpoints like:
+- Add/complete/remove multiple items at once
+- Adding multiple items with the same Name but different specifications
+- You can work with a unique identifier for an item even before adding it to a list, just use uuid4 to generate a random uuid!
+
+Usage examples:
+
+### Add an item 
+When adding an item, the `itemId` is required, `spec` and `uuid` are optional. If you need a unique identifier before adding an item, you can just generate a uuid4.
+
+```python
+item = {
+  "itemId": "Cilantro",
+  "spec": "fresh",
+  "uuid": "xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxx"
+}
+await bring.batch_update_list(
+  lists[0]['listUuid'],
+  item,
+  BringItemOperation.ADD)
+```
+### Updating an items specification
+
+When updating an item, use ADD operation again. The `itemId` is required and the item `spec` will be added/updated on the existing item on the list. For better matching an existing item (if there is more than one item with the same `itemId`), you can use it's unique identifier `uuid`.
+
+```python
+item = {
+  "itemId": "Cilantro",
+  "spec": "dried",
+  "uuid": "xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxx"
+}
+await bring.batch_update_list(
+  lists[0]['listUuid'],
+  item,
+  BringItemOperation.ADD)
+```
+
+### Multiple items
+
+```python
+# multiple items can be passed as list of items
+items = [{
+  "itemId": "Cilantro",
+  "spec": "fresh",
+  "uuid": "xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxx"
+},
+{
+  "itemId": "Parsley",
+  "spec": "dried",
+  "uuid": "xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxx"
+}]
+await bring.batch_update_list(
+  lists[0]['listUuid'],
+  items,
+  BringItemOperation.ADD)
+```
+
+### Add multiple items with the same name but different specifications.
+
+When adding items with the same name the parameter `uuid` is required, otherwise the previous item will be matched by `itemId` and it's specification will be overwritten
+
+```python
+items = [
+  {
+    "itemId": "Cilantro",
+    "spec": "100g, dried",
+    "uuid": "xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxx"
+  },
+    {
+    "itemId": "Cilantro",
+    "spec": "fresh",
+    "uuid": "xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxx"
+  }
+]
+await bring.batch_update_list(

```

Comment:
Do we want to raise if someone does not respect the required `uuid` it this case? Or just let the user figure it out by himself?

Reply:
nah, uuid is optional, it just is required for this special case, were one wants to have different items with the same name. In this example if uuid is omitted, it would simply result in the item "Cilantro, 100g, dried" being created and instantly its specification beeing changed to "fresh". But raising would also mean, that the uuid cannot be omitted intentionally and also e.g. make the usage of the method save_item more complicated.